perception_dataset/deepen/deepen_to_t4_converter.py

```diff
@@ -138,7 +139,9 @@ def _convert_occulusion_to_visibility(self, name: str) -> str:
         else:
             return "full"
 
-    def _format_deepen_annotation(self, label_dicts: List[Dict[str, Any]]):
+    def _format_deepen_annotation(
+        self, label_dicts: List[Dict[str, Any]], camera_index: Dict[str, int] = None
```

Comment:
@ktro2828 
Right. Since this is only used [here](https://github.com/tier4/tier4_perception_dataset/blob/d5ca86ab475ffaaed118c3684ec648a7c2d62db4/perception_dataset/deepen/deepen_to_t4_converter.py#L63), there are no cases where this option argument be None.

Reply:
Ah, wait. If there is no images in the dataset (lidar only case), camera_index might be None.

Reply:
@miursh Thanks, make sense. Then it is reasonable to declare `camera_index: Optional[Dict[str, int]] = None` in the variable of method.

Also,  I think we need to add validation to confirm `camera_index` is not `None` before line 264.

Reply:
@ktro2828 Thx! 36d77522ec5383dcd2badc7b6db6ee709b920697